en-US/Design.xml

```diff
@@ -721,6 +721,34 @@ $ vi myFile.txt
 		</para>
 
 	</section>
+
+	<section id="making-recommendations">
+	 <title>Making Recommendations</title>
+		<para>
+		When making a recommendation, the preferred verbiage is <emphasis role="bold">"Red&nbsp;Hat recommends..."</emphasis> instead of the common but indirect "It is recommended...".
+		Recommendations can include best practices, recommended practices, and product-specific suggestions.
+		Refer to <xref linkend="avoiding-confusing-language"/> for information on using the terms "best practices" and "recommended practices" in Red&nbsp;Hat documentation.
+	  </para>
+    <para>
+	   <emphasis role="strong">Example 1</emphasis>
```

Comment:
`<emphasis role="strong">` ?
You might be able to make use of `<example>` elements here. Have a look at how IBM does their correct/incorrect examples; maybe we can borrow their idea.

Reply:
Updated to use <example>